example/build.gradle

```diff
@@ -22,7 +22,7 @@ android {
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     testCompile 'junit:junit:4.12'
-    compile 'com.android.support:appcompat-v7:23.1.1'
-    compile 'com.android.support:design:23.1.1'
+    compile 'com.android.support:appcompat-v7:23.3.0'
+    compile 'com.android.support:design:23.3.0'
```

Comment:
Update to support libs here: https://github.com/vimeo/vimeo-networking-java/pull/84/commits/76a81bae43fad8494775a395635131b3edf9be16